### DISCUSSION TITLE
Patterns for emails and/or letters

### DISCUSSION THREAD

**Discussion by daniel-ac-martin** [UKHomeOffice/design-system#377] [Ideas]
Is this something the design system should cover? (There have been some calls for it.)

If so, what would the guidance be?

(This came from a ticket raised elsewhere by @MarionDownie.)

**Comment by daniel-ac-martin** [comment]
Apparently this is on the 'content patterns' backlog.

**Comment by Sarah-Lombardi** [comment]
I have captured a few examples and put them in the content pattern library.

Refugee integration loans letter: [Refugee Integration Loan letter.docx](https://homeofficegovuk.sharepoint.com/:w:/s/UCDcommunityHub/EbFhm5ndkg1DuXr24Ed3FcYBCOUM6SbOX0KgYJ6veqSe3A?e=ryYzDj)
Notice of intended prosecution letter: [Notice of intended prosecution letter.docx](https://homeofficegovuk.sharepoint.com/:w:/s/UCDcommunityHub/EfRtWAPWtgNBt5qEvDOu8VcBbztPIdNLiYzylJEjsaYSAg?e=yEd6a5)

Refugee integration loans email: [Refugee Integration Loan email.docx](https://homeofficegovuk.sharepoint.com/:w:/s/UCDcommunityHub/EdkCQ4fmrt1ErcgqLZmyNqcBKWfAvrsiLqAdbiv199lITg?e=Yh5cTP)
UKVI email template: [UKVI email templates.docx](https://homeofficegovuk.sharepoint.com/:w:/s/UCDcommunityHub/Ed-c3UAq_EpIgWiUNCTXH2EBkGdNPg9kMEUCusTO9Y5d0w?e=iyMvfO)
COP email template: [COP email template.docx](https://homeofficegovuk.sharepoint.com/:w:/s/UCDcommunityHub/EVcu1EpPvlhEn5GWWTUjbEQBKz7fvt7auKyFh0k-zN6v7A?e=tA2n6J)

**Reply by Piriya-Mahendra-Pordes** [reply]
Do you have any findings relating to length, eg would you be able to specify a rough word count for a service letter? Single side of A4?

**Reply by Sarah-Lombardi** [reply]
We tried not to be too prescriptive in the pattern as it depends on the type of letter (eg asylum refusal letters can run to several pages but they also contain a write up of the decision itself so you will never get that to under 1 page of A4). 'Keep it to 1 page' did come up as a theme in the workshop though we didn't commit ourselves to it. In general I think its about writing concisely. What that looks like will differ depending on what the letter covers. But other advice we did put forward is if your content does extend beyond 1 page then you should use page numbers and make it clear if content is front and back of the sheet of paper so users don't miss anything.

**Comment by Sarah-Lombardi** [comment]
Planning a workshop to share examples and discuss best practice as well.

**Comment by Sarah-Lombardi** [comment]
Mural board from workshop here: https://app.mural.co/t/ctodesignresearch0795/m/ctodesignresearch0795/1657794861638/d84b7d98da77d433d6889fefde2f8032245e92b3?sender=u1332f5ac9b0b4c5cf37f7848

**Comment by Sarah-Lombardi** [comment]
Our findings were:

- Lots of specific factors influencing how we can do letters and emails, including access to existing letters and emails, being able to make changes, timings of when we're included, and involvement from stakeholders like policy, legal and strategic comms
- It works best when content designers are included early on - UKV&I are a good example of this and could be used as models for what good looks like
- A lot of what good looks like is the same as the advice on the service manual, but there might be value in sharing a redacted example of what we think is an ideal model

### Letters
Themes for good letters are:

- Use the recipients full name
- Use plain English, especially for non-native speakers
- Keep to 1 page if you can
- If the letter continues beyond 1 page, tell them (eg page 1 of 5)
- Make the outcome you are reporting and next steps the most prominent things - do not bury them under standard paragraphs, they should be on page 1
- Point to GOV.UK for guidance, rather than writing it out again in the letter
- Prioritise the content
- Make the reference number clear
- Give clear contact details
- Work with policy and legal on standard paragraphs
- Avoid jargon and acronyms
- Carefully balance customisation (which meets staff needs and stops them having to store tens or hundreds of templates) against user needs (the content could become a bit repetitive or disjointed)
- Send by email if they have stated it as a preference

Outstanding questions:

- In what cases do we need 'on behalf of secretary of state'?
- When would we add a translation?
- When should and shouldn't we send letters as PDFs?

### Emails
Themes for good emails are:

- Bring in content designers at first draft
- Give their reference number and explain how they use it
- Spell out URLs to reassure users - but consider UX of writing out a long weird URL
- Don't use short URLs - unless officially granted by GDS
- Hierarchy based on information need
- Use their full name
- Use Plain English, especially for non-native English speakers
- Make the main decision or outcome the most prominent thing
- Call to action should be highlighted
- Use Notify if you can - it is more accessible and easier to copy and translate

Outstanding questions:

- We have heard that some users need guidance in one place rather than linking away to other websites - which services was this on and what do we know about it?
- In what circumstances do we offer a translation within the email?
- In what circumstances are there messages from ministers within the body text (as with Ukraine family visas)?
- Home Office users can get a lot of spam emails and texts so how do we build trust? For example, say something like we will never ask for X in our emails or texts and have standard formats and language?

**Reply by MarionDownie** [reply]
'We have heard that some users need guidance in one place rather than linking away to other websites - which services was this on and what do we know about it?' - I believe this was the Ukraine visas letter

**Comment by Sarah-Lombardi** [comment]
Taken from Horizon guidance on writing letters.

> All letters should:
> 
> - address the issues raised by the correspondent – not what we think they ought to be asking.  In particular you should avoid simple copy and paste exercises of standard lines and consider exactly what is being asked
> - be mindful of the circumstances of the correspondent- across the department we deal with a range of sensitive issues, think about how the letter will be received and remember there is a real person at the end of the process
> - use clear, uncomplicated language that mirrors the current government's position and avoid jargon
> - ensure the policy position outlined in the draft response remains up to date.  We understand this can be difficult in some fast-moving areas, however you are the experts in your field and we rely on you to let us know if lines on your policy change before a response has been issued
> - avoid spelling mistakes, grammatical errors and typos.  This sounds like a simple one but comes up a lot.  When you’re focused on a piece on work it’s easy to see what you think is there and miss the little errors. Do check and double check or get a colleague with a fresh pair of eyes to take a look for you
> - take on board feedback.  If amendments have been made to earlier drafts, please do bear that in mind and ensure those amendments are incorporated into any new drafts on the same topic as no-one enjoys giving the same feedback repeatedly

**Comment by Sarah-Lombardi** [comment]
This is where I've got to. I think a content model would work quite well as it leaves enough scope for some of the variations across the department. I'll write it up and hand over to someone before I go on leave.

<img width="505" alt="Screenshot 2022-09-09 at 15 55 45" src="https://user-images.githubusercontent.com/39190312/191212709-da5b6238-1854-44fa-9e72-fc96efd930cc.png">
<img width="514" alt="Screenshot 2022-09-09 at 15 55 52" src="https://user-images.githubusercontent.com/39190312/191212133-a16ad678-11c6-4254-b331-42b0fe147d23.png">
<img width="276" alt="Screenshot 2022-09-09 at 15 56 00" src="https://user-images.githubusercontent.com/39190312/191212234-d56097af-9d6b-422d-92a9-28e9fd242482.png">
<img width="275" alt="Screenshot 2022-09-09 at 15 56 06" src="https://user-images.githubusercontent.com/39190312/191212202-2230208d-3f57-4a3f-acde-266290fcf68e.png">

**Reply by Williams-Leith** [reply]
Do we have to use a passive construction in the headline?

**Reply by Sarah-Lombardi** [reply]
No not at all - I think what matters is that it’s clear, immediate, meaningful and easy to relate to. And if we do use pronouns, make sure it’s clear who’s being referred to. But you’re right, better to avoid the passive. In this case I was front loading the thing they’d applied for so they could easily relate it to something, but there are other ways we could do it. I did consider making it more of a outcome ie “you can study in the UK from (date to date)” but that lost the word visa and when I considered it in the negative it seemed too final “you cannot study in the UK”. What do you think?

**Comment by tomyems** [comment]
Draft guidance: https://633aabbca7b24e4143196396--hods.netlify.app/patterns/send-a-letter

**Comment by tomyems** [comment]
Updated guidance: https://63402bb447e213111ee70022--hods.netlify.app/patterns/send-a-letter

**Comment by tomyems** [comment]
Fixed typo: https://63402f5df8e580141894c955--hods.netlify.app/patterns/send-a-letter

**Comment by tomyems** [comment]
Investigate making the page title 'Send users a letter - Home Office Design System'

**Reply by tomyems** [reply]
Update the 'updates section on the home page'

**Reply by tomyems** [reply]
Updated page title and the updates on the home page: https://634987dea2e38b004f5aa17a--hods.netlify.app/patterns/send-a-letter

**Comment by tomyems** [comment]
Change the 'element: description' to be 'element: summary' and enhance the text to be:

Summarise the outcome or action:

- Lists can help ...

**Reply by tomyems** [reply]
Updated to: https://63510d6e8cddf900b234eb51--hods.netlify.app/patterns/send-a-letter

(also made the capitalisation of 'First name and last name' consistent to 'Dear first name and last name,')

**Comment by tomyems** [comment]
To match GOV.UK Notify letters, moved reference number below main action. Also for readability grouped the 'header' items.

Updated to: https://63513998886e1c1f978d1836--hods.netlify.app/patterns/send-a-letter

**Comment by Sarraounia** [comment]
Great work on this pattern @Sarah-Lombardi and @tomyems. One thought from me is around the 'What to include in a letter' section. It says 'each letter should have' but it's not clear to me if the blocks are being suggested as mandatory, like 'do this in all cases' or 'do what suits your service.' Are there any blocks that are immovable? Not sure if they're a mixture of what should be on every template and what you could include. Also unsure if the 'additional content' is a nice-to-have rather than expected to be in every letter.

**Reply by tomyems** [reply]
I suspect it's hard to be definitive and prescriptive. I think possibly all mandatory, but maybe there's some letters with good reason not to include certain parts. Hence the 'should'. - There might be a better way of phrasing this to improve clarity.

additional content is not always necessary, but if there is additional content it should go at the end (often) on an another page

**Comment by tomyems** [comment]
From accessibility input, add guidance for using a sans-serif font, such as Arial, font size 12. Align to one side of the page, left for left to right languages, right for right to left languages.

No italics and advise against block capitals.

**Comment by tomyems** [comment]
Updated with clarification about additional letter content and accessibility input: https://637cb6ba731dbc03e943fb37--hods.netlify.app/patterns/send-a-letter

**Comment by tomyems** [comment]
Added semantic structure: https://637d0469f3661000608af927--hods.netlify.app/patterns/send-a-letter

**Comment by tomyems** [comment]
Clarified only for posted letters: https://638090c15737096c0e176c29--hods.netlify.app/patterns/send-a-letter

**Comment by tomyems** [comment]
Minor content update: https://638092e327e28361a9f48c53--hods.netlify.app/patterns/send-a-letter

**Comment by tomyems** [comment]
On emails, should there be a full stop after a hyperlink, so:

If you need help, contact us at: [https://www.gov.uk/contact-ukvi-inside-outside-uk](https://www.gov.uk/contact-ukvi-inside-outside-uk)

or

If you need help, contact us at: [https://www.gov.uk/contact-ukvi-inside-outside-uk](https://www.gov.uk/contact-ukvi-inside-outside-uk).

**Reply by tomyems** [reply]
Confirmed that as users may try and copy and paste the link, do not include a full stop

**Comment by Sarah-Lombardi** [comment]
Email guidance drafted by Sarah Downes. Content crit planned for 11 July.

**Comment by DeborahStevenson** [comment]
Attached draft material from Sarah Downes. Content crit completed in July. Sent for wider feedback.

[Send an Email Pattern.docx](https://github.com/user-attachments/files/16945629/Send.an.Email.Pattern.docx)
![Write an email draft pattern (1)](https://github.com/user-attachments/assets/a950acae-662b-4fef-8f13-2060057a6bfe)

**Comment by clairefellows** [comment]
The pattern has been updated following feedback and the branch has been created on GitHub. Please review and provide any further feedback. 
![screencapture-localhost-8080-patterns-send-an-email-2025-03-21-10_07_38](https://github.com/user-attachments/assets/e793ab4f-e69e-4f06-a086-21e1e9c4e611)

**Comment by clairefellows** [comment]
Following discussion on the Design Working Group and confirmation this page has had an internal 2i, it was agreed to push this pattern live. Any further feedback welcome.

https://67dd4f150c7176034d543682--hods.netlify.app/patterns/send-an-email
